tests/unit/test_views.py

```diff
@@ -131,7 +131,7 @@ def sendReferenceBasesList(self, id_=None):
         return response
 
     def test404sReturnJson(self):
-        path = utils.applyVersion('/doesNotExist')
+        path = '/doesNotExist'
```

Comment:
Maybe we could get some more examples here? I.e. things which have versions included, things which don't, things which are prefixes of legal URLs, legal URLS that have a suffix (.e.g., GET /variantsets/id/should404) . We should be testing a whole pile of different things here to make sure they 404.